sgkit/tests/test_window.py

```diff
@@ -64,13 +64,22 @@ def test_window():
     assert not has_windows(ds)
     ds = window(ds, 2, 2)
     assert has_windows(ds)
+    np.testing.assert_equal(ds[window_contig].values, [0, 0, 0, 0, 0])
     np.testing.assert_equal(ds[window_start].values, [0, 2, 4, 6, 8])
     np.testing.assert_equal(ds[window_stop].values, [2, 4, 6, 8, 10])
 
     with pytest.raises(MergeWarning):
         window(ds, 2, 2)
 
 
+def test_window__multiple_contigs():
+    ds = simulate_genotype_call_dataset(n_variant=15, n_sample=1, n_contig=3)
```

Comment:
It'd be nice to test this with n_contig=n_variant, so we test the degenerate case.

Also be nice to see what happens when n_variant=1.

Reply:
Added in https://github.com/pystatgen/sgkit/pull/372/commits/399bd045acb7f99f63a91ddce9fae82bf623b280